lib/course.rb

```diff
@@ -1 +1,24 @@
+class Course
+  attr_reader :name, :capacity, :students
 
+  def initialize(name,capacity)
```

Comment:
```suggestion
  def initialize(name, capacity)
```
Convention is to put a space after the comma